### PR TITLE
[WIP - don't merge yet] Added stuff for MySQL and debezium

### PR DESCRIPTION
**Pull request by devinbost**
I'd like some early feedback before I go too deep down the path I took in this PR. 

(Notice most of the logic in deploy_mysql_k8s.sh)

The basic flow of this is:
mysql in k8s deployment (with local Pulsar) -> debezium source -> app that sends data to Astra topic -> AstraDB sink (haven't done this part yet)

Note that this code is based on my other branch that I still need to update based on Yabin's feedback.

### CODE REVIEW

File: cluster_deploy/ext_tools/mysql/deploy_mysql_k8s.sh [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+# TODO: Make sure we have built project before we call this script since it requires Jars to be available, so make sure we're calling this function in the right place during deployment.
+
+# Expecting first parameter to be the scenario name:
+scnName=$1
+
+
+source "${PULSAR_WORKSHOP_HOMEDIR}/_bash_utils_/utilities.sh"
+
+scnHomeDir="${PULSAR_WORKSHOP_HOMEDIR}/scenarios/${scnName}"
```
Comment (yabinmeng) [inline cluster_deploy/ext_tools/mysql/deploy_mysql_k8s.sh:37]: Anything under `cluster_deploy` should be scenario agnostic. In my opinion, the script here should be only used for deploying MySQL in K8s. I don't recommend we put scenario specific stuff here

Having said that, we probably need to extend `deployScenario.sh` to add support for deploying source/sink connectors.

File: application_code/client_apps/java/debezium_replicator_producer_consumer/src/main/java/com/example/pulsarworkshop/PulsarConsumerFullCfg.java [hunk 2]
```diff
@@ -0,0 +1,186 @@
+package com.example.pulsarworkshop;
```
Comment (yabinmeng) [inline application_code/client_apps/java/debezium_replicator_producer_consumer/src/main/java/com/example/pulsarworkshop/PulsarConsumerFullCfg.java:1]: Please use a different file name other than `PulsarConsumerFullCfg.java` that is more relevant with your use case.

**Review by yabinmeng — CHANGES_REQUESTED** [review]

File: application_code/client_apps/java/debezium_replicator_producer_consumer/src/main/java/com/example/pulsarworkshop/PulsarConsumerFullCfg.java (hunk 2 above)
Comment (devinbost) [reply application_code/client_apps/java/debezium_replicator_producer_consumer/src/main/java/com/example/pulsarworkshop/PulsarConsumerFullCfg.java]: Good catch. This was an oversight.

**Comment by yabinmeng** [comment]
Won't consider in the new framework

**Pull request closed without merging**